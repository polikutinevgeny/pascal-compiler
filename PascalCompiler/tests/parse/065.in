var
   i: integer;
   a: double;
begin
   for i := 0 to 10 do
      for i := 3 downto 0 do
         a := 34.5;
end.