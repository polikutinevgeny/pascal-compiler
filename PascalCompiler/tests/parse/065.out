program 
  integer;
  real;
  char;
  i: integer ;
  a: real ;
└─For
  ├─i
  ├─0
  ├─10
  └─For
    ├─i
    ├─3
    ├─0
    └─Assignment statement
      ├─a
      └─34.5
